exporter/exporterhelper/retry_sender.go

```diff
@@ -117,7 +117,6 @@ func (rs *retrySender) send(ctx context.Context, req Request) error {
 			trace.WithAttributes(rs.traceAttribute, attribute.Int64("retry_num", retryNum)))
 
 		err := rs.nextSender.send(ctx, req)
-		rs.logger.Info("Exporting finished.", zap.Error(err))
```

Comment:
Should we switch it to a Debug? Or would it be too noisy even for that?

Reply:
Debug is probably ok

Reply:
imo it's not useful. The error is logged if present a few lines below, and I don't see value in outputting a debug statement that an export is done, when we also have tracing and span events collected around those lines.